### PR TITLE
Translate thinking-in-react.md

### LINKED ISSUE #77: Thinking in React
I would take this as my first page!

### PR DESCRIPTION
**Pull request by rammba**
I've added translation for `thinking-in-react.md`. Feel free to add suggestions and comments.

Closes #77

### CODE REVIEW

File: src/content/learn/thinking-in-react.md [hunk 1]
```diff
@@ -437,15 +438,21 @@ td {
...
+Primetićete da promena forme još uvek ne radi. Pojavio se problem u konzoli u sandbox-u iznad koji objašnjava zašto:
 
 <ConsoleBlock level="error">
 
 You provided a \`value\` prop to a form field without an \`onChange\` handler. This will render a read-only field.
 
 </ConsoleBlock>
 
-In the sandbox above, `ProductTable` and `SearchBar` read the `filterText` and `inStockOnly` props to render the table, the input, and the checkbox. For example, here is how `SearchBar` populates the input value:
+<ConsoleBlock level="error">
```
Comment (rammba) [inline src/content/learn/thinking-in-react.md:449]: I've added new console block to show translated message also. Is this correct way, or we can use something else.

Examples on other languages:
image

image

image

**Review by lxmarinkovic — COMMENTED** [review]
lgtm, but one thing before I merge this. I have decided not to translate the word "component" as one of react terms and keep it as it is or hyphenated when necessary. The logic behind this is to avoid any ambiguities when we have the Serbian word "komponenta" necessary in translation since it would keep straight (as this document says) thinking in react. Pleease let me know your thoughts on this :D

**Comment by rammba** [comment]
For me it's more fluid to use `komponenta` instead of `component`. I would like to read these docs in the same way as I would say something to my colleague everyday.
For example: I would rather say `"Trebali bi da pomerimo onaj input u Button komponentu."`, instead of `"Trebali bi da pomerimo onaj input u Button component."`
I know that component is widely known term in many frameworks, not only React, but I would stick with translated term.

**Comment by lxmarinkovic** [comment]
> For me it's more fluid to use `komponenta` instead of `component`. I would like to read these docs in the same way as I would say something to my colleague everyday.
> 
> For example: I would rather say `"Trebali bi da pomerimo onaj input u Button komponentu."`, instead of `"Trebali bi da pomerimo onaj input u Button component."`
> 
> I know that component is widely known term in many frameworks, not only React, but I would stick with translated term.

In my opinion you got the point but "Trebalo bi da pomerimo onaj input u Button component-u" would sound the same but avoiding possible ambiguities. I both ways the word is Latin not English, not Serbian… If you more ideas why this should be better we can discuss it on discord :D

**Pull request merged**